src/runtime/runtimeClient.ts

```diff
@@ -118,6 +118,19 @@ export abstract class RuntimeClient {
         actions.push(new ActionImplementations["+="]("event", e, a, v, "event", scopes));
       }
       this.evaluation.executeActions(actions);
+    } else if(data.type === "result" && (data.database === "server" || data.database === "browser-session")) {
+      // @TODO: should we allow databases other than server at some point?
```

Comment:
Yeah, we need to figure out how the extensibility story and policy story interact to give us this if we think it's important. This isn't blocking but it merits a research task.